tests/OSPSuite.Core.IntegrationTests/OSPSuite.Core.IntegrationTests.csproj

```diff
@@ -63,6 +63,9 @@
     <None Update="Data\EventsAppKeywordReplacement.pkml">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="Data\MultipleDosing_0_8_16.csv">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
     <None Update="Data\MultipleDosing_0_8_16.xls">
```

Comment:
Let's remove the excel files if they are not used anymore

Reply:
I did already ;)
https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/pull/1599/commits/04b49097a26bc91d2d1046b168da4e05c866551f#diff-941dec3a96d741e2508f082ae94091684e864546b347ecafdc39f3dca68af427
https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/pull/1599/commits/04b49097a26bc91d2d1046b168da4e05c866551f#diff-f0a484c7362aa86842f38acac2348d5cb163d20878540e3ab3becf6091d3664a